1053/javascript.js

```diff
@@ -0,0 +1,33 @@
+readline = require("readline");
+
+process.stdin.resume();
+process.stdin.setEncoding('utf-8');
+
+let inputString = '';
+let currentLine = 0;
+
+process.stdin.on('data', function(inputStdin) {
+    inputString += inputStdin;
+});
+
+function readLine() {
+    return inputString[currentLine++];
+}
+
+process.stdin.on('end', function() {
+    inputString = inputString.split('\n');
+    run_it();
+});
+
+function solution(singleCase){
```

Comment:
this method should be like
```
function solution(a, b, c){
}
```

because on the problem statements, there are three inputs per case.

Reply:
Fixed the function arguments as informed